aten/src/ATen/core/DeprecatedTypeProperties.cpp

```diff
@@ -0,0 +1,24 @@
+#include <ATen/core/DeprecatedTypeProperties.h>
+
+#include <ATen/core/LegacyTypeDispatch.h>
+#include <ATen/core/Type.h>
+
+namespace at {
+
+Tensor DeprecatedTypeProperties::unsafeTensorFromTH(void * th_pointer, bool retain) const {
```

Comment:
how did you gather this collection of functions to add?

Reply:
also, is the plan here that the implementations of these functions will move here?  Because if dispatch type loses scalars, the implementations probably won't work as is, right?

Reply:
I think the only function that people would actually use off of Type is copy. The others I moved because the tests access them from at::CPU(), which now returns DeprecatedTypeProperties.

I'm going to keep these calls here for now. Copy I'm moving to native, and i'll just call an equivalent here instead. The others don't actually need to come off of Type right away because they don't use dtype.